### PR TITLE
Adds a Meta library with an eval function for interpreting code inline.

### PR DESCRIPTION
**Pull request by gsmaverick**
This PR adds a simple `Meta.eval(_)` function that accepts a string to evaluate inline. The biggest limitation right now is that you can't define new names inside the evalled source and then use those names afterwards because the compiler assumes those names aren't defined in the source and can't generate the correct bytecode for them.

### CODE REVIEW

**Comment by zeckalpha** [comment]
:+1:

File: src/vm/wren_meta.h [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#ifndef wren_meta_h
+#define wren_meta_h
+
+#include <stdio.h>
+
+#include "wren.h"
+#include "wren_common.h"
+#include "wren_value.h"
+#include "wren_vm.h"
```
Comment (munificent) [inline src/vm/wren_meta.h]: I don't think `wren_value` or `wren_vm` is used here.

File: src/vm/wren_meta.h [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#ifndef wren_meta_h
+#define wren_meta_h
+
+#include <stdio.h>
```
Comment (munificent) [inline src/vm/wren_meta.h]: Can this be removed?

**Comment by munificent** [comment]
> you can't define new names inside the evalled source and then use those names afterwards

That's the behavior I expect, I think. I believe the REPL works the same way if the variable isn't capitalized.

**Comment by gsmaverick** [comment]
@munificent updated to remove the unnecessary header files.

**Comment by munificent** [comment]
:metal:

**Pull request merged**